### DISCUSSION TITLE
Idea: Create a "Continuous Mode" view for tags but only the paragraph they appear in -

### DISCUSSION THREAD

**Discussion by smiledahling** [gasparschott/obsidian-continuous-mode#20] [Ideas]
Example: 

Search by tag
Results appear in the search window
Right click -> Continuous Mode -> Open Proximity Search Result in New Split (R, L, U, D) 

You can use some sort of Boolean logic proximity operations like NEAR, WITHIN, or SENTENCE. 

Reference: https://www.scribbr.com/working-with-sources/boolean-operators/

**Comment by gasparschott** [comment]
Hmm, if I understand your suggestion correctly, it seems to me that this sort of thing is beyond the scope of the plugin, at least in its current form. It might be better handled by a search plugin (I haven't used it, but perhaps https://github.com/ryanjamurphy/vantage-obsidian). Let me know if you have any other ideas. Thanks.
